README.md

```diff
@@ -44,58 +44,121 @@ You will need to find a permanent location for it and add it to your PATH.
 
 # Use
 
-This CLI is called `svcat` on the command line. See below for a description 
-of the commands that `svcat` offers.
+This CLI is called `svcat` on the command line. Run `svcat -h` to see the available
+commands.
 
-## Commands for `ClusterServiceBroker`s
+Below are some common tasks made easy with `svcat`. The example output assumes that [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure/) is installed on the cluster.
 
-To list all the brokers in the cluster:
+## Find brokers installed on the cluster
 
 ```console
-svcat get brokers
+$ svcat get brokers
 ```
 
-To get the status of an individual broker:
+## List available service classes
 
 ```console
-svcat get broker <broker name>
+$ svcat get classes
+            NAME                      DESCRIPTION                             UUID
++--------------------------+--------------------------------+--------------------------------------+
+  azure-rediscache           Azure Redis Cache                0346088a-d4b2-4478-aa32-f18e295ec1d9
+  azure-storage              Azure Storage                    2e2fc314-37b6-4587-8127-8f9ee8b33fea
+  azure-aci                  Azure Container Instance         451d5d19-4575-4d4a-9474-116f705ecc95
+  azure-cosmos-document-db   Azure DocumentDB provided by     6330de6f-a561-43ea-a15e-b99f44d183e6
+                             CosmosDB and accessible via
+                             SQL (DocumentDB), Gremlin
+                             (Graph), and Table (Key-Value)
+                             APIs
+  azure-servicebus           Azure Service Bus                6dc44338-2f13-4bc5-9247-5b1b3c5462d3
+  azure-eventhub             Azure Event Hub Service          7bade660-32f1-4fd7-b9e6-d416d975170b
+  azure-cosmos-mongo-db      MongoDB on Azure provided by     8797a079-5346-4e84-8018-b7d5ea5c0e3a
+                             CosmosDB
+  azure-mysqldb              Azure Database for MySQL         997b8372-8dac-40ac-ae65-758b4a5075a5
+                             Service
+  azure-postgresqldb         Azure Database for PostgreSQL    b43b4bba-5741-4d98-a10b-17dc5cee0175
+                             Service
+  azuresearch                Azure Search                     c54902aa-3027-4c5c-8e96-5b3d3b452f7f
+  azure-keyvault             Azure Key Vault                  d90c881e-c9bb-4e07-a87b-fcfe87e03276
+  azure-sqldb                Azure SQL Database Service       fb9bc99e-0aa9-11e6-8a8a-000d3a002ed5
```

Comment:
can you make a note here to indicate that this output would come up if you had OSBA installed